### DISCUSSION TITLE
can't run yarn install, how to config git lfs

### DISCUSSION THREAD

**Discussion by LeZhang13** [foxglove/community#57] [Q&A]
Hi, I'm a beginner about front-end develop, I install the node v16.14.2, yarn V1.22.18, Git Lfs, then I want run yarn install ,but show the error. 
`"Foxglove Error: Please configure Git LFS ( https://git-lfs.github.com/ ) then run `git lfs pull` before running yarn."
`Then I run 'git lgs pull' , show error:
```
Could not pull

Errors logged to '/home/tedzhang/Foxglove/studio-main/.git/lfs/logs/20220701T173655.986169578.log'.
Use `git lfs logs last` to view the log.
```
I have no idea and coundn't find any guild to tell me how to configure Git LFS...
Eventhough I run yarn -V, also show:
`"Foxglove Error: Please configure Git LFS ( https://git-lfs.github.com/ ) then run `git lfs pull` before running yarn."
`
![image](https://user-images.githubusercontent.com/89899260/176872315-0eff09cd-fc0a-48c7-a110-4d4ca9f46043.png)
![image](https://user-images.githubusercontent.com/89899260/176872410-d82a1f51-dab7-4eab-bc39-e3c535df56bb.png)
![image](https://user-images.githubusercontent.com/89899260/176872481-e371ea3b-6137-48ee-a602-6444ceb179f7.png)

**Comment by jtbandes** [comment]
It looks like you downloaded the source code as a zip file. Instead you will need to use Git to [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository):

```
git clone https://github.com/foxglove/studio.git
```
